main_test.go

```diff
@@ -1,3 +1,112 @@
 package main
 
-// Пишите тесты в этом файле
+import (
+	"fmt"
+	"testing"
+)
+
+//тест для функции generateRandomElements
+
+func TestGenerateRandomElement(t *testing.T) {
+
+	//тестовые данные
+	size1 := 100_000_001
+	size2 := -10
+	size3 := 0
+
+	//проверка на 0
+	fmt.Printf("Тест №1: если аргумент равен %d, ожидаем пустой слайс\n", size3)
+	data := generateRandomElements(size3)
+	if len(data) != 0 {
```

Comment:
В тестах есть много тестовых функций, везде if нужно убрать, вот пример
```suggestion
	require.Lenf(t, data, 0, "Ожидаемый результат: 0, получен размер %d", len(data))
```